doc/readme/EXAMPLES.md

```diff
@@ -35,13 +35,16 @@ Alternatively, you see [INTERFACES.md](INTERFACES.md) on how to run them manuall
 For details, see `CMakeLists.txt` in `examples/meta/` for details, `generate.py` and `translate.py` in `examples/meta/generator/`.
 
 ## Adding new examples
-It is extremely simple to add a new example: simple create another `*.sg` file.
-We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
+It is extremely simple to add a new example: simple create another `*.sg.in` file in the `examples/meta/src/*/` directory. For example, if you are porting `kernel-linear.py` then you should create `linear.sg.in` under `examples/meta/src/kernel/`. To create corresponding `.sg` file it is essential to run `cmake .. && make` and then port the examples using `make meta_examples`, please ensure `BUILD_META_EXAMPLES=ON` is set. We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
```

Comment:
```suggestion
It is extremely simple to add a new example: simply create another `*.sg.in` file in the `examples/meta/src/*/` directory.
For example, if you are porting `kernel-linear.py` then you should create `examples/meta/src/**kernel/linear.sg.in**`. To create the corresponding `.sg` file (which will be located in the build directory) run `cmake .. && cmake --build . --target meta_examples`. Please ensure `BUILD_META_EXAMPLES=ON` is set. 
We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the shogun's meta language. For further details have a look at #3555.
```

Reply:
@NanuSai when writing docs you shouldn't assume that people are using `make`, some of us use `ninja` or the visual studio build system. The generic way to do it with cmake is `cmake --build . -- target TARGET`.